### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by Njoe00**
fixing header styling to match more closely to website copy.

### CODE REVIEW

File: src/components/SliderButton.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default function WalletImageSlider({
   }
 
   return (
-    <div className="flex flex-col justify-center w-full z-40 absolute bottom-0">
+    <div className="flex-nowrap flex-col justify-center w-full float-left z-40 absolute bottom-0">
```
Comment (MKCMMSK) [inline src/components/SliderButton.tsx]: This seems strange to me, you're using flex box and also using float here. From my understanding, flexbox can't use floats.

File: src/components/SliderButton.tsx [hunk 2]
```diff
@@ -36,9 +36,9 @@ export default function WalletImageSlider({
           value={sliderValue}
           onChange={handleEventChange}
         />
-        <li className="flex w-full absolute text-center top-14 left-8">
+        <li className="flex flex-nowrap truncate w-full absolute text-center items-center top-14 left-6 flex-row">
           <span
-            className="flex text-slider-colour-full font-bold text-end text-sm justify-end z-10"
+            className="inline w-max text-slider-colour-full font-bold items-center text-end text-sm z-10"
             style={{
               width: filledWidthCalucation(),
```
Comment (MKCMMSK) [inline src/components/SliderButton.tsx]: I believe there will be conflicts once you merge your other PR. Need to resolve

**Review by MKCMMSK — CHANGES_REQUESTED** [review]
Couple of things to consider

File: src/components/SliderButton.tsx (hunk 1 above)
Comment (Njoe00) [reply src/components/SliderButton.tsx]: leave behind from changes. Removed float-left.

File: src/components/SliderButton.tsx (hunk 2 above)
Comment (Njoe00) [reply src/components/SliderButton.tsx]: changed feature/main-image-slider to match current code to avoid conflicts

**Review by MKCMMSK — APPROVED** [review]

**Pull request merged**